packages/core/lib/commands/test/run.js

```diff
@@ -116,9 +139,13 @@ module.exports = async function (options) {
 
     // configuredNetwork will spread only when it is defined and ignored when undefined
     ganacheOptions = { ...ganacheOptions, port, ...configuredNetwork };
+
+    // Sanitize the ganache options if required
```

Comment:
Small nit. This comment is redundant, as your choice in function name and variable serves as adequate documentation.